Mathlib/Algebra/Group/Defs.lean

```diff
@@ -68,7 +68,7 @@ See the module documentation near the macro for more details.
 class HSMul (α : Type u) (β : Type v) (γ : outParam (Type w)) where
   /-- `a • b` computes the product of `a` and `b`.
   The meaning of this notation is type-dependent, but it is intended to be used for left actions. -/
-  hSMul : α → β → γ
+  hsmul : α → β → γ
```

Comment:
```suggestion
  hSMul : α → β → γ
```
because this matches `HMul.hMul`, which is not something we have the power to change.